src/resolve/resolved-dns-cache.c

```diff
@@ -670,6 +667,7 @@ static bool rr_eligible(DnsResourceRecord *rr) {
 int dns_cache_put(
                 DnsCache *c,
                 DnsCacheMode cache_mode,
+                uint64_t cache_size,
```

Comment:
you don't make use of this param

Reply:
the size limit should be propagated into the DnsCache object when it is first initialized. Not when we add something to it.